lib/ui/huge_listview/lazy_loading_gallery.dart

```diff
@@ -132,6 +149,8 @@ class _LazyLoadingGalleryState extends State<LazyLoadingGallery> {
   void dispose() {
     _reloadEventSubscription.cancel();
     _currentIndexSubscription.cancel();
+    widget.selectedFiles.removeListener(() {});
+    _shouldSelectAll.removeListener(() {});
```

Comment:
Where are we adding listener for `_shouldSelectAll`?

Reply:
At line 274

Reply:
Have moved it.